src/pages/task/taskTabs/testsTable/TestsTableCore.tsx

```diff
@@ -44,6 +44,12 @@ export const TestsTableCore: React.FC = () => {
   const queryVariables = getQueryVariables(search, resourceId);
   const { cat, dir, pageNum, limitNum } = queryVariables;
 
+  if (cat === undefined) {
```

Comment:
we should check this condition and call updateQueryParams in a useEffect. that lets us be more intentional about calling updateQueryParams since the useEffect will run only when the specified value change

Reply:
Good catch